src/main/java/io/jenkins/plugins/audit/CustomHTMLLayout.java

```diff
@@ -0,0 +1,412 @@
+    private static final String DEFAULT_TITLE = "Log4j Log Messages";
+    private static final String DEFAULT_CONTENT_TYPE = "text/html";
+
+    private final long jvmStartTime = ManagementFactory.getRuntimeMXBean().getStartTime();
+
+    // Print no location info by default
+    private final boolean locationInfo;
+    private final String title;
+    private final String contentType;
+    private final String font;
+    private final String fontSize;
+    private final String headerSize;
+
+    /**Possible font sizes */
+    public static enum FontSize {
+        SMALLER("smaller"), XXSMALL("xx-small"), XSMALL("x-small"), SMALL("small"), MEDIUM("medium"), LARGE("large"),
+        XLARGE("x-large"), XXLARGE("xx-large"),  LARGER("larger");
+
+        private final String size;
+
+        private FontSize(final String size) {
+            this.size = size;
+        }
+
+        public String getFontSize() {
+            return size;
+        }
+
+        public static FontSize getFontSize(final String size) {
+            for (final FontSize fontSize : values()) {
+                if (fontSize.size.equals(size)) {
+                    return fontSize;
+                }
+            }
+            return SMALL;
+        }
+
+        public FontSize larger() {
+            return this.ordinal() < XXLARGE.ordinal() ? FontSize.values()[this.ordinal() + 1] : this;
+        }
+    }
+
+    private CustomHTMLLayout(final boolean locationInfo, final String title, final String contentType, final Charset charset,
+            final String font, final String fontSize, final String headerSize) {
+        super(charset);
+        this.locationInfo = locationInfo;
+        this.title = title;
+        this.contentType = addCharsetToContentType(contentType);
+        this.font = font;
+        this.fontSize = fontSize;
+        this.headerSize = headerSize;
+    }
+
+    /**
+     * For testing purposes.
+     */
+    public String getTitle() {
+        return title;
+    }
+
+    /**
+     * For testing purposes.
+     */
+    public boolean isLocationInfo() {
+        return locationInfo;
+    }
+
+    @Override
+    public boolean requiresLocation() {
+        return locationInfo;
+    }
+
+    private String addCharsetToContentType(final String contentType) {
+        if (contentType == null) {
+            return DEFAULT_CONTENT_TYPE + "; charset=" + getCharset();
+        }
+        return contentType.contains("charset") ? contentType : contentType + "; charset=" + getCharset();
+    }
+
+    /**
+     * Formats as a String.
+     *
+     * @param event The Logging Event.
+     * @return A String containing the LogEvent as HTML.
+     */
+    @Override
+    public String toSerializable(final LogEvent event) {
+        final StringBuilder sbuf = getStringBuilder();
+
+        sbuf.append(Strings.LINE_SEPARATOR).append("<tr>").append(Strings.LINE_SEPARATOR);
+
+        sbuf.append("<td>");
+        sbuf.append(event.getTimeMillis() - jvmStartTime);
```

Comment:
I'm not sure why the original layout did this. Could you change this to do something like:

```suggestion
        sbuf.append(DateTimeHelper.formatDateISO(event.getTimeMillis()));
```

Reply:
I couldn't find the package for DateTimeHelper class. I've replaced it with different class with same output formatting.